doc/classes/Label.xml

```diff
@@ -127,6 +127,7 @@
 		<theme_item name="outline_size" data_type="constant" type="int" default="0">
 			Text outline size.
 			[b]Note:[/b] If using a font with [member FontFile.multichannel_signed_distance_field] enabled, its [member FontFile.msdf_pixel_range] must be set to at least [i]twice[/i] the value of [theme_item outline_size] for outline rendering to look correct. Otherwise, the outline may appear to be cut off earlier than intended.
+			[b]Note:[/b] Using a large value is not recommended. The upper limit depends on the font being used, but this value should typically not be greater than 100.
```

Comment:
What's a "large value" in this context depends on the size of the font itself. I'd suggest wording it in a relative manner:

> Using a value that is larger than half the font size isn't recommended, as the font outline may fail to be fully closed in this case.